.travis.yml

```diff
@@ -1,17 +1,18 @@
-# This is necessary until https://github.com/travis-ci/travis-ci/issues/3120 is
-# fixed
-sudo: required
+# container-based build
+sudo: false
```

Comment:
Any reason not to use container based builds for Finatra, T-S, Scrooge, and util as well?